tgui/packages/tgui/interfaces/APC.js

```diff
@@ -197,32 +197,29 @@ const ApcContent = (props, context) => {
               <Button
                 icon={data.coverLocked ? 'lock' : 'unlock'}
                 content={data.coverLocked ? 'Engaged' : 'Disengaged'}
-                selected={data.coverLocked}
                 disabled={locked}
                 onClick={() => act('cover')}
               />
             }
           />
           <LabeledList.Item
-            label="Night Shift Lighting"
+            label="Emergency Lighting"
             buttons={
               <Button
                 icon="lightbulb-o"
-                content={data.nightshiftLights ? 'Enabled' : 'Disabled'}
-                selected={data.nightshiftLights}
-                onClick={() => act('toggle_nightshift')}
+                content={data.emergencyLights ? 'Enabled' : 'Disabled'}
+                disabled={locked}
+                onClick={() => act('emergency_lighting')}
               />
             }
           />
           <LabeledList.Item
-            label="Emergency Light Fallback"
+            label="Night Shift Lighting"
             buttons={
               <Button
-                icon={data.coverLocked ? 'lock' : 'unlock'}
-                content={data.emergencyLights ? 'Engaged' : 'Disengaged'}
-                selected={data.emergencyLights}
-                disabled={locked}
-                onClick={() => act('emergency_lighting')}
+                icon="lightbulb-o"
+                content={data.nightshiftLights ? 'Enabled' : 'Disabled'}
+                onClick={() => act('toggle_nightshift')}
```

Comment:
Any reason for these being swapped? No big deal but just so I understand

Reply:
Discussed on Discord. As a note, the label has been updated for the emergency lights.